README.md

```diff
@@ -1,12 +1,13 @@
 # Jupyter Kernel Gateway
 
-A [JupyterApp](https://github.com/jupyter/jupyter_core/blob/master/jupyter_core/application.py) that implements the HTTP and Websocket resources of `/api/kernels` using [jupyter/notebook](https://github.com/jupyter/notebook), [jupyter/jupyter_client](https://github.com/jupyter/jupyter_client), and [jupyter/jupyter_core](https://github.com/jupyter/jupyter_core) as libraries. Launches kernels in its local process/filesystem space. Can be containerized and scaled out by a cluster manager (e.g., [tmpnb](https://github.com/juputer/tmpnb)). Intended to be accessed by [jupyter-js-services](https://github.com/jupyter/jupyter-js-services) and similar web-friendly clients.
+A [JupyterApp](https://github.com/jupyter/jupyter_core/blob/master/jupyter_core/application.py) that implements different execution models for Notebooks. The first method is for accessing HTTP and Websocket resources of `/api/kernels` using [jupyter/notebook](https://github.com/jupyter/notebook), [jupyter/jupyter_client](https://github.com/jupyter/jupyter_client), and [jupyter/jupyter_core](https://github.com/jupyter/jupyter_core). This model launches kernels in its local process/filesystem space. Can be containerized and scaled out by a cluster manager (e.g., [tmpnb](https://github.com/juputer/tmpnb)). API endpoints can be accessed by [jupyter-js-services](https://github.com/jupyter/jupyter-js-services) and similar web-friendly clients. Another model is also provided to expose cells from a notebook as HTTP endpoints.
```

Comment:
The http / websocket model doesn't have to do with execution of notebooks. Suggest:

```
A [JupyterApp](https://github.com/jupyter/jupyter_core/blob/master/jupyter_core/application.py) that implements different APIs and protocols for accessing Jupyter kernels such as:
```

and then proceed to list them. Refer to your new section with a link too where appropriate so people know to look down further to understand it. Move the generic bits after the list of specifics (e.g., *This model launches kernels in its local process/filesystem space.* is true for **both** API+protocol options currently implemented).